UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Controllers/RadioNavigationalWarningsAdminController.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using UKHO.MaritimeSafetyInformation.Common.Logging;
+using UKHO.MaritimeSafetyInformation.Common.Models.RadioNavigationalWarning.DTO;
+using UKHO.MaritimeSafetyInformation.Web.Services;
+
+namespace UKHO.MaritimeSafetyInformation.Web.Controllers
+{
+    public class RadioNavigationalWarningsAdminController : BaseController<RadioNavigationalWarningsAdminController>
+    {
+        private readonly IRnwRepository _iRnwRepository;
+        private readonly ILogger<RadioNavigationalWarningsAdminController> _logger;
+
+        public RadioNavigationalWarningsAdminController(IHttpContextAccessor contextAccessor,
+                                                        ILogger<RadioNavigationalWarningsAdminController> logger,
+                                                        IRnwRepository iRnwRepository) : base(contextAccessor, logger)
+        {
+            _iRnwRepository = iRnwRepository;
```

Comment:
Please don't inject repository directly into controller instead add service layer in between for business logic to maintain separation of concerns. Refer changes made into branch "Feature/46539 list all rnw records"

Reply:
done required changes